bin/rspec

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
```

Comment:
I've always thought checking in binstubs is a [recommended best practice](https://github.com/rbenv/rbenv/wiki/Understanding-binstubs)

> (*You are encouraged to check these binstubs in the project's version control so your colleagues might benefit from them.*)

But maybe not for libraries?